Vic2ToHoI4/Source/HOI4World/Diplomacy/HoI4AIStrategy.h

```diff
@@ -0,0 +1,41 @@
+#ifndef HoI4AISTRATEGY_H_
+#define HoI4AISTRATEGY_H_
+
+
+
+#include "../../V2World/Ai/AIStrategy.h"
+
+
+
+namespace HoI4
+{
+
+class AIStrategy
+{
+  public:
+	explicit AIStrategy(std::string strategyType): type(std::move(strategyType)) {}
+
+	AIStrategy(const Vic2::AIStrategy& oldStrategy):
+		 type(std::move(oldStrategy.getType())), id(oldStrategy.getID()), value(oldStrategy.getValue())
```

Comment:
std::move is extraneous here.

Reply:
It looks like when you use this you immediately replace the id with the HoI4 tag. Just take that tag as a parameter instead,